src/main/java/helpy/Helpy.java

```diff
@@ -0,0 +1,157 @@
+package helpy;
+
+import helpy.exceptions.IllegalDescriptionException;
+import helpy.task.Deadline;
+import helpy.task.Event;
+import helpy.task.Task;
+import helpy.task.Todo;
+
+import java.util.Scanner;
+import java.util.ArrayList;
+
+public class Helpy {
+    public static final String HORIZONTAL_LINE = "______________________\n";
+
+    public static void greetUser() {
+        String name = "░▒█░▒█░▒█▀▀▀░▒█░░░░▒█▀▀█░▒█░░▒█\n" +
+                "░▒█▀▀█░▒█▀▀▀░▒█░░░░▒█▄▄█░▒▀▄▄▄▀\n" +
+                "░▒█░▒█░▒█▄▄▄░▒█▄▄█░▒█░░░░░░▒█░░\n";
+        System.out.println(HORIZONTAL_LINE
+                + "Greetings, I am\n" + name);
+        System.out.println("How can I help you?\n" + HORIZONTAL_LINE);
+    }
+
+    public static void listTasks(ArrayList<Task> taskList) {
+        int label = 1;
+
+        System.out.print(HORIZONTAL_LINE);
+        System.out.println("These are the tasks in your list:");
+        for (Task task : taskList) {
+            System.out.print(label + ".");
+            System.out.println(task);
+            label++;
+        }
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    public static void printMessage(String message) {
+        System.out.print(HORIZONTAL_LINE);
+        System.out.println(message);
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    public static void markTask(Task task, String commandStartsWith) {
+
+        System.out.print(HORIZONTAL_LINE);
+
+        if (commandStartsWith.equals("mark")) {
+            task.setDone(true);
+            System.out.println("Good job! I've marked this task as done:");
+        } else {
+            task.setDone(false);
+            System.out.println("Ok, this task has been marked as not done yet:");
+        }
+
+        System.out.println("\t" + task);
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    public static void addMessage(ArrayList<Task> taskList) {
+        int numOfTasks = taskList.toArray().length;
+        Task theTask = taskList.get(numOfTasks - 1);
+        System.out.print(HORIZONTAL_LINE);
+        System.out.println("Ok I just added: ");
+        System.out.println("\t" + theTask);
+
+        if (numOfTasks == 1) {
+            System.out.println("There is 1 task in the list.");
+        } else {
+            System.out.println("There are " + numOfTasks
+                    + " tasks in the list.");
+        }
+        System.out.println(HORIZONTAL_LINE);
+    }
+
+    public static void processCommand(String command, ArrayList<Task> taskList) {
```

Comment:
The processCommand method is rather long, try to abstract out the processing implementations of each command.